docs/conf/scalar-loki-stack-custom-values.yaml

```diff
@@ -53,3 +53,27 @@ promtail:
                 - __meta_kubernetes_pod_uid
                 - __meta_kubernetes_pod_container_name
               target_label: __path__
+        # -- the `scalar-admin-for-kubernetes` job scrapes all the logs of Scalar Admin for Kubernetes Pods
```

Comment:
Sorry that I'm not familiar with the Loki configurations, and it might not be directly related to this PR; let me ask several questions.

- Do we need to scrape all the logs in every job execution?
- scraped results are stored anywhere persistent?  Or if Loki logs are removed or rotated, are they gone?

Anyway, we should keep the paused duration persistently independent from the Loki logs lifecycle, and I'm wondering if we can achieve that.

Reply:
@feeblefakie 

I had a discussion with Yokoyama-san and confirm the feasibility.
In short, Scalar Manager will periodically (maybe per minute) get paused durations from Loki and preserve them in the ConfigMap. So, it doesn't matter how users configure Loki's retention or storage. Scalar Manager provides paused duration by itself.

Reply:
@supl So, the answers to the questions are both No?

Reply:
If the answer to the first question is yes, it might be an issue. For example, if the job scrapes all the logs in every execution, and users configure a long retention period, I think it might make the job execution take quite long and expensive.

Reply:
@feeblefakie 

Sorry for my late response.

> Do we need to scrape all the logs in every job execution?

In the comment `the `scalar-admin-for-kubernetes` job scrapes all the logs of Scalar Admin for Kubernetes Pods`, the `all` is for pods.

Maybe it's better to changed the sentence to `the `scalar-admin-for-kubernetes` job scrapes the logs of all Scalar Admin for Kubernetes Pods`

The Promtail job is a progressive process (tail) that keeps shipping new updated logs to Loki. So it's no problem even it has been running and shipping many logs for a very long time.

> scraped results are stored anywhere persistent? Or if Loki logs are removed or rotated, are they gone?

1. Loki has no retention by default unless users explicitly configure it. If users specify a retention, the Loki remove the logs older than the retention from its storage.
2. Loki's default storage is file-based similar to `/var/logs`. If we don't use a permanent volume, or change the storage to a cloud service, for example, DynamoDB, we will loss the logs when Loki server is restarted.